src/block/icon-list/style.scss

```diff
@@ -35,3 +35,12 @@
 	margin-left: var(--stk-alignment-margin-left);
 	margin-right: var(--stk-alignment-margin-right);
 }
+
+//Suggestion from blocksy, prevents.
```

Comment:
Please also adjust the comment, no need to mention that it's a suggestion from blocksy